openpaas-james/apps/distributed-ldap/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+version: '3'
+
+networks:
+  emaily:
+    driver: bridge
+
+services:
+  james:
+    depends_on:
+      - elasticsearch
+      - cassandra
+      - tika
+      - rabbitmq
+      - s3
+      - ldap
+    image: linagora/openpaas-james-distributed-ldap
+    container_name: james
+    hostname: james.local
+    ports:
+      - "80:80"
+      - "25:25"
+      - "110:110"
+      - "143:143"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+      - "8000:8000"
+    networks:
+      - emaily
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.3.2
+    environment:
+      - discovery.type=single-node
+    networks:
+      - emaily
+
+  cassandra:
+    image: cassandra:3.11.3
+    ports:
+      - "9042:9042"
+    networks:
+      - emaily
+
+  tika:
+    image: apache/tika:1.24
+    networks:
+      - emaily
+
+  rabbitmq:
+    image: rabbitmq:3.8.3-management
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    networks:
+      - emaily
+
+  s3:
+    image: zenko/cloudserver:8.2.6
+    container_name: s3.docker.test
+    environment:
+      - SCALITY_ACCESS_KEY_ID=accessKey1
+      - SCALITY_SECRET_ACCESS_KEY=secretKey1
+      - S3BACKEND=mem
+      - LOG_LEVEL=trace
+      - REMOTE_MANAGEMENT_DISABLE=1
+    networks:
+      - emaily
+
+  ldap:
+    image: preprovisioned_ldap
```

Comment:
Should we potentially upload that on our linagora repos? If you have a better name btw... :)

Reply:
Cant you use `dinkel/openldap:latest` and use a volume to mount a `/etc/ldap/prepopulate/prepop.ldif` file in it so that no action is needed?

Reply:
Can try